anvil/anvil-annotations/src/main/java/com/duckduckgo/anvil/annotations/ContributesWorker.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.anvil.annotations
+
+import kotlin.reflect.KClass
+
+annotation class ContributesWorker(
```

Comment:
nit: we should add target and retention to help the compiler know about lifecycle of the annotation
```suggestion
@Target(AnnotationTarget.CLASS)
@Retention(AnnotationRetention.RUNTIME)
annotation class ContributesWorker(
```